### DISCUSSION TITLE
Bot is selling runes and not using the pickit i set!

### DISCUSSION THREAD

**Discussion by Zaidore** [blizzhackers/kolbot#421] [Q&A about d2bot-with-kolbot]
hi.

i started using kolbot again recently but running into problems, i have 3 bots up and running perfectly, in the beginning they would pick up runes and what not, after 1 day or so they stopped picking them up and just selling them.

one bot is on lead.dbj and 2 on follow.dbj all have their own legit keys. i tried to adjust the kolton.lib so it would pick up everything and i dont use the other file wich is also deleted from the char config.

can someone help me out with this i have been searching on google but cant find this same problem, even after a complete reinstall today it still wont work, i would really apreciate your help!

**Comment by bmarc94** [comment]
hi ! 

I'm a bit late =)
Since the ladder reset i try to customize the bot, so can try to help.

You should not have any problems if you have this line one ALL your char config

`Config.PickitFiles.push("myCustomPickOrAnytFile.nip");`

with that command inside

`[Name] == ElRune`

By default, in kolton.nip you there's this line:  `[name] >= istrune && [name] <= zodrune`
This means : pick  everything from Ist to Zod

You can change istrune with a lower runes of your choice, so your bot gonna

I would says it's no matter if it happens, but be ware to not have duplicate line like

`[name] == istrune # # [maxquantity] == 1`



Well if everything is checked, share your seller char config, and your Kolton.nip. I'll try to analyse it

(i'm off tonight for  10 days more or less)
